libretro.c

```diff
@@ -219,6 +217,7 @@ bool retro_load_game(const struct retro_game_info *info)
 
 void retro_unload_game(void)
 {
+   lutro_deinit();
```

Comment:
Unsure if this is correct... Unloading the game should unload the game, not the core.

Reply:
I think in this case lutro needs content loaded before it can deinit the core. I think the simplest way is to move `lutro_deinit()`, but alternatively the this could be fixed in RetroArch by preventing the call to `retro_deinit` if content was not loaded. However I think that might not be correct and may break other cores in trivial ways which do not crash when `retro_deinit` is called without content loaded.

Since only two cores that I know of need this fix I think its better fixed in the cores, but maybe the crash can be avoided somewhere in `lutro_deinit()` instead without requiring to move it?